Project.toml

```
@@ -5,17 +5,11 @@ version = "0.1.1"
 
 [deps]
 CEnum = "fa961155-64e5-5f13-b03f-caf6b980ea82"
-ColorTypes = "3da002f7-5984-5a60-b8a6-cbb66c0b333f"
-FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
-FixedPointNumbers = "53c48c17-4a7d-5ca2-90c5-79b7896eea93"
 ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
 libpng_jll = "b53b4c65-9356-5827-b1ea-8c7a1a84506f"
 
 [compat]
 CEnum = "^0.2"
-ColorTypes = "^0.9"
-FileIO = "^1.2"
-FixedPointNumbers = "^0.7"
 ImageCore = "^0.8"
```

more specifically, by [ImageCore 0.8.1](https://github.com/JuliaImages/ImageCore.jl/releases/tag/v0.8.1)

```suggestion
ImageCore = "^0.8.1"
```

FYI, the caret syntax is used as default and thus not recommended to use according to [BlueStyle](https://github.com/invenia/BlueStyle#package-version-specifications)

---

Ah, thanks, TIL :)